src/Avalonia.Base/Threading/AvaloniaScheduler.cs

```diff
@@ -46,7 +46,7 @@ IDisposable PostOnDispatcher()
                     {
                         composite.Add(action(this, state));
                     }
-                }, DispatcherPriority.DataBind);
+                }, DispatcherPriority.Background);
```

Comment:
@grokys Need your input about this one since I'm not familiar with AvaloniaScheduler usage. Is it only used by bindings?  If it is, it's wrong to change the priority, but if it's not then having it higher than Render _might_ cause those "window got closed inside `[NSView updateLayer]` callback" issues.